src/ddmd/dsymbol.d

```diff
@@ -269,7 +269,7 @@ extern (C++) class Dsymbol : RootObject
         return false;
     }
 
-    final bool isAnonymous()
+    bool isAnonymous()
```

Comment:
So you made this function virtual, but didn't reflect it in c++ headers?

Reply:
I thought I had the C++ headers covered but apparently not.

Reply:
See https://github.com/dlang/dmd/pull/7385